### ISSUE TITLE
KeyError after calling bot.gateway.session.user

### ISSUE THREAD

**Issue by dos320** [vivinano/MudaeAutoBot#116]
![image](https://user-images.githubusercontent.com/44009661/137382117-d669c053-d769-4444-b4b1-e4620e66fba5.png)

No idea why this happens, but it seems like an issue with discum. I think it may be on my end... any solutions?

**Comment by vivinano** [comment]
Looks like an invalid discord token

**Comment by dos320** [comment]
I've been using the same user account the whole time, and it worked before, like a month ago

**Comment by vivinano** [comment]
Can I see the screenshot of this log details as that is weird

**Comment by vivinano** [comment]
Hmm all I can think of is try to print out user and if it's empty I think this might be a discum dev issue 

Did anything change in terms of using like any updates cause I haven't pushed any code changes if it was working previously

**Comment by dos320** [comment]
yeah it gives the same KeyError, no matter where in on_message it is. Weird since opening the gateway before printing should solve this problem, which you do

**Comment by vivinano** [comment]
Yeah I would still ask the dev over at discum just to see why it is returning null cause that is a function on their end
